=CRASH REPORT==== 25-Oct-2013::17:55:25 ===
  crasher:
    initial call: rabbit_reader:init/4
    pid: <0.1304.2>
    registered_name: []
    exception exit: channel_termination_timeout
      in function  rabbit_reader:wait_for_channel_termination/2 
      in call from rabbit_reader:handle_exception/3 
      in call from rabbit_reader:terminate/2 
      in call from rabbit_reader:handle_other/2 
      in call from rabbit_reader:mainloop/2 
      in call from rabbit_reader:run/1 
      in call from rabbit_reader:start_connection/7 
    ancestors: [<0.1301.2>,rabbit_tcp_client_sup,rabbit_sup,<0.31483.1>]
    messages: [emit_stats]
    links: []
    dictionary: [{{ch_pid,<0.1314.2>},{1601,#Ref<0.0.23.59065>}},
                  {credit_blocked,[]},
                  {{channel,1601},
                   {<0.1314.2>,{method,rabbit_framing_amqp_0_9_1}}}]
    trap_exit: true
    status: running
    heap_size: 987
    stack_size: 27
    reductions: 14152
  neighbours:

=SUPERVISOR REPORT==== 25-Oct-2013::17:55:25 ===
     Supervisor: {<0.1301.2>,rabbit_connection_sup}
     Context:    shutdown_error
     Reason:     channel_termination_timeout
     Offender:   [{pid,<0.1304.2>},
                  {name,reader},
                  {mfa,{rabbit_reader,start_link,
                                      [<0.1303.2>,<0.1302.2>,
                                       #Fun<rabbit_heartbeat.2.69784259>]}},
                  {restart_type,intrinsic},
                  {shutdown,4294967295},
                  {child_type,worker}]


